packages/daemon/src/services/index.ts

```diff
@@ -537,6 +544,54 @@ export const voidTx = async (
   await voidTransaction(mysql, hash, addressBalanceMap);
   await markUtxosAsVoided(mysql, dbTxOutputs);
 
+
+  // CRITICAL: Unspent the inputs when voiding a transaction
```

Comment:
chore: typo. This is also present in other points in this PR.
```suggestion
  // CRITICAL: Unspend the inputs when voiding a transaction
```

Reply:
Fixed, thanks